mmr/mmr_test.go

```diff
@@ -58,11 +60,9 @@ func testMMR(count uint32, proofElem []uint32) error {
 	}
 
 	mmrTree.Commit()
-
 	result := proof.Verify(root)
-
 	if !result {
-		return err
+		return fmt.Errorf("error verifying root")
```

Comment:
since you are not adding any wildcard expressions, this would be better with `return errors.New("error verifying root")`